api/stats.js

```diff
@@ -1,31 +1,73 @@
 import { config } from './../config.js';
 import { SnowflakeUtil, MessageEmbed } from "discord.js";
 
+const removeParentheses = word => word.replace(/\(|\)/g, '');
```

Comment:
It would be good to move this method into and utils directory.

Reply:
That's a great idea, however, i like doing that only when we know we use it in more than one place, which is not the case here.